src/opendr/perception/pose_estimation/hr_pose_estimation/HighResolutionLearner.py

```diff
@@ -0,0 +1,992 @@
+            checkpoint_name = "checkpoint_iter_" + str(self.checkpoint_load_iter) + ".pth"
+            checkpoints_folder = os.path.join(self.parent_dir, '{}_checkpoints'.format(self.experiment_name))
+            full_path = os.path.join(checkpoints_folder, checkpoint_name)
+            try:
+                checkpoint = torch.load(full_path, map_location=torch.device(self.device))
+            except FileNotFoundError as e:
+                e.strerror = "File " + checkpoint_name + " not found inside checkpoints_folder, " \
+                                                         "provided checkpoint_load_iter (" + \
+                             str(self.checkpoint_load_iter) + \
+                             ") doesn't correspond to a saved checkpoint.\nNo such file or directory."
+                raise e
+            if not silent and verbose:
+                print("Loading checkpoint:", full_path)
+
+            # Loads weights in self.model from checkpoint
+            # if self.from_mobilenet:  # TODO see todo on ctor
+            #     load_from_mobilenet(self.model, checkpoint)
+            # else:
+            load_state(self.model, checkpoint)
+        elif self.model is None:
+            raise AttributeError("self.model is None. Please load a model or set checkpoint_load_iter.")
+
+        self.model = self.model.eval()  # Change model state to evaluation
+        self.model.to(self.device)
+        if "cuda" in self.device:
+            self.model = self.model.to(self.device)
+            if self.half:
+                self.model.half()
+
+        if self.multiscale:
+            self.scales = [0.5, 1.0, 1.5, 2.0]
+
+        coco_result = []
+        num_keypoints = Pose.num_kpts
+
+        pbar_eval = None
+        if not silent:
+            pbarDesc = "Evaluation progress"
+            pbar_eval = tqdm(desc=pbarDesc, total=len(data), bar_format="{l_bar}%s{bar}{r_bar}" % '\x1b[38;5;231m')
+
+        img_height = data[0]['img'].shape[0]
+
+        if img_height == 1080 or img_height == 1440:
+            offset = 200
+        elif img_height == 720:
+            offset = 50
+        else:
+            offset = 0
+
+        for sample in data:
+            file_name = sample['file_name']
+            img = sample['img']
+            h, w, _ = img.shape
+            max_width = w
+            kernel = int(h / self.first_pass_height)
+            if kernel > 0:
+                pool_img = HighResolutionPoseEstimationLearner.Pooling(self, img, kernel)
+
+            else:
+                pool_img = img
+
+            perc = 0.3  # percentage around cropping
+
+            thresshold = 0.1  # thresshold for heatmap
+
+            # ------- Heatmap Generation -------
+            avg_pafs = HighResolutionPoseEstimationLearner.first_pass(self, self.model, pool_img)
+            avg_pafs = avg_pafs.astype(np.float32)
+
+            pafs_map = cv2.blur(avg_pafs, (5, 5))
+            pafs_map[pafs_map < thresshold] = 0
```

Comment:
`thresshold` -> `threshold`
```suggestion
            threshold = 0.1  # threshold for heatmap

            # ------- Heatmap Generation -------
            avg_pafs = HighResolutionPoseEstimationLearner.first_pass(self, self.model, pool_img)
            avg_pafs = avg_pafs.astype(np.float32)

            pafs_map = cv2.blur(avg_pafs, (5, 5))
            pafs_map[pafs_map < threshold] = 0
```